libsql-ffi/bundled/SQLite3MultipleCiphers/src/sqlite3.c

```diff
@@ -85673,8 +85673,8 @@ void blobSpotFree(BlobSpot *pBlobSpot);
 
 /*
  * Accessor for node binary format
- * - v1 format is the following:
- *   [u64 nRowid] [u16 nEdges] [node vector] [edge vector] * nEdges [trash vector] * (nMaxEdges - nEdges) ([u64 legacyField] [u64 edgeId]) * nEdges
+ * - default format is the following:
+ *   [u64 nRowid] [u16 nEdges] [2 byte padding] [node vector] [edge vector] * nEdges [trash vector] * (nMaxEdges - nEdges) ([u32 unused] [f32 distance] [u64 edgeId]) * nEdges
```

Comment:
I must be missing something. If nRowid + nEdges takes 64 + 16 = 80 bits and nodeMetadataSize for V3 is 64 + 64 = 128 bits then the padding should be 48 bits (6 bytes) instead of 2 bytes (=16 bits), no?

Reply:
Yes, this was a bug in comment. Fixed it.